README .R

```diff
@@ -0,0 +1,28 @@
+
+library(dplyr)
+library(readr)
+library(stringr)
+library(sbtools)
+library(whisker)
+
+source("1_fetch/src/fetch_data.R")
+source("2_process/src/process_data.R") 
+source("3_visualize/src/visualize_data.R")
+
+project_output_dir <- '1_fetch/out'
+mendota_file <- file.path(project_output_dir, 'model_RMSEs.csv')
+
+fetch_data(mendota_file)
+
+
+
+project_output_dir <- '2_process/out'
+eval_data <- load_data(mendota_file = mendota_file)
+summary_model(mendota_file = eval_data)
+
+
+project_output_dir <- '3_visualize/out'
```

Comment:
Similar to an above comment: I would like to see this (new) value of `project_output_dir` formally passed (as an argument) to `plot_results()`

It might also be good to pass this value as an argument to `diagnostic_log_file()`, depending on whether you decide to move that function to another phase or not.

Reply:
this comment still stands (in review number 2)